src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double base1;
+    private double base2;
+    private double height;
+
+    IsoscelesTrapezoid(Color color, double base1, double base2, double height) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
+        return ((base1 + base2) / 2) * height;
+    }
+
+    @Override
+    public void drawFigure() {
+        System.out.println("Figure: isosceles trapezoid, area: " + calculateArea()
```

Comment:
Avoid using `System.out.println()` directly in the `drawFigure()` method. It's better to return the string representation and let the caller decide how to display or utilize the returned string.